cleverpush/src/main/java/com/cleverpush/stories/models/Preview.java

```diff
@@ -0,0 +1,89 @@
+
+package com.cleverpush.stories.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class Preview implements Serializable {
+
+    @SerializedName("publisher")
+    @Expose
+    private String publisher;
+    @SerializedName("publisherLogoSrc")
+    @Expose
+    private String publisherLogoSrc;
+    @SerializedName("posterPortraitSrc")
+    @Expose
+    private String posterPortraitSrc;
+    @SerializedName("publisherLogoWidth")
+    @Expose
+    private Integer publisherLogoWidth;
+    @SerializedName("publisherLogoHeight")
+    @Expose
+    private Integer publisherLogoHeight;
+    @SerializedName("posterLandscapeSrc")
+    @Expose
+    private String posterLandscapeSrc;
+    @SerializedName("posterSquareSrc")
+    @Expose
+    private String posterSquareSrc;
+
+    public String getPublisher() {
+        return publisher;
+    }
+
+    public void setPublisher(String publisher) {
+        this.publisher = publisher;
+    }
+
+    public String getPublisherLogoSrc() {
+        return publisherLogoSrc;
```

Comment:
please return this.publisherLogoSrc, as you want to reference to the class scope variable (make it more descriptive)